ansible/configs/ansible-automation-platform/post_software.yml

```diff
@@ -53,39 +53,17 @@
       agnosticd_user_info:
         msg: |
             To Access VScode UI via browser:
-            VScode UI URL: https://bastion.{{ agnosticd_domain_name }}
-            VScode UI Password: {{ hostvars[groups['bastions'][0]]['tower_admin_password'] }}
+                URL: https://bastion.{{ agnosticd_domain_name }} 
```

Comment:
can you fix the trailing space to pass the CI tests?

Reply:
Argh, again forgot to configure VSCode... ;)

Done.